src/components/Input/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import Search from '@material-design-icons/svg/round/search.svg';
+import Clear from '@material-design-icons/svg/round/cancel.svg';
+
+import { Input, Props } from '../Input';
```

Comment:
But if you can't pass a label here or errormessage, it won't be passed to the input either. I thought because of the sentence 'All variants, states, sizes and types are supported' in the ticket, these things should be passed on.

Reply:
I did change it like you suggested, but storybook still shows those options. I can't change it, I tried all kinds of things... Most common you use argTypes to set it to disabled, but it is aware that those props are not longer part of searchInput, so I can't do that :') 

Anyway, hope it's okay like this